sof-dev-defconfig

```diff
@@ -10,6 +10,7 @@ CONFIG_SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE=y
 #CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST will be obsolete when IPC_FLOOD_TEST_CLIENT is merged
 CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST=y
 CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST_CLIENT=m
+CONFIG_SND_SOC_SOF_DEBUG_PROBES_CLIENT=m
```

Comment:
can you keep the old definition until you changes get merged?

Reply:
@plbossart we're not doing anything to do with probes in our daily tests or CI today. besides with the new changes SND_SOC_SOF_DEBUG_PROBES will be selected automatically (which is what was there originally)

Reply:
ok, thanks for the precisions